doc/ansible/dev/apb_migration_guide.md

```diff
@@ -155,7 +155,7 @@ would become:
 
 ```yaml
 - name: Create bind credential secret
-  k8s:
+  community.kubernetes.k8s:
```

Comment:
Note that there's a different way (which I prefer) to upgrade to using collections, and that is to add a `collections` keyword on the play level, listing the collections you use, then leave the module invocations the same. Otherwise you end up writing things like `community.kubernetes.k8s` over and over and over, getting carpal tunnel.

Example:

```yaml
- hosts: hosts_here
  collections:
    - community.kubernetes
    - operator_sdk.util
  tasks:
    - k8s:
      ...
```

Reply:
I put in the FQCN here and in other examples because they're outside of the context of a playbook and it didn't feel like an appropriate spot to explain collections and tell users to add those to their playbooks, though I did update the scaffolded code to use the collection play keyword when generating the playbook. Since it's possible to generate just a role, no playbook, and AFAIK there is no way to pass the collection keyword when running a role directly, I do think it makes sense to include the FQCN outside of playbook examples. Thoughts?

Reply:
> AFAIK there is no way to pass the collection keyword when running a role directly

I haaaaaaaaate that. I'm hoping to campaign for that not to be the case because I absolutely hate the idea of having to type out a FQCN every time you type in a module in Ansible.